#Common Configuation for fake agents
ros_queue_size: 1
request_target_channel: "interest_point"
temperature_measurement_channel: "fake_agent_temperature_measurement"
temperature_update_channel: "fake_agent_temperature_update_channel"
report_gps_location_channel: "Fake_GPS"
fake_moving_duration_threshold_s: 40
fake_distance_threshold_s: 0.01
nagivate_loop_rate: 10
observation_noise_std: 0.1
# To create groundtruth distribution
ground_truth_mu: [8, 9, 
                  7, 10, 
                  1, 0]

ground_truth_sig: [0.1, 0.5, 
                   0.2, 0.8, 
                   0.7, 0.4]
ground_truth_num_gaussian: 3